tools/osx/crosstool/BUILD.toolchains

```diff
@@ -23,6 +23,14 @@ OSX_TOOLS_CONSTRAINTS = {
         "@platforms//os:osx",
         "@platforms//cpu:x86_64",
     ],
+    "maccatalyst_x86_64": [
```

Comment:
Given the CPUs in java code we might need this to be `catalyst_x86_64` without the `mac` prefix.